Recast/Cells/SeriesGridCollectionViewCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SeriesCollectionViewCell.swift
+//  Recast
+//
+//  Created by Jaewon Sim on 9/16/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class SeriesGridCollectionViewCell: UICollectionViewCell {
+    private var seriesImageView: UIImageView!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        seriesImageView = UIImageView()
+        seriesImageView.layer.cornerRadius = 8
```

Comment:
use the same corner radius ... or even better make a helper UIImageView subclass or function that makes image views with the same corner radius?